nixos/modules/installer/sd-card/sd-image-aarch64.nix

```diff
@@ -39,6 +39,17 @@
         # Supported in newer board revisions
         arm_boost=1
 
+        [cm4]
+        kernel=u-boot-rpi4.bin
+        enable_gic=1
+        armstub=armstub8-gic.bin
+        disable_overscan=1
+
+        # Enable host mode on the 2711 built-in XHCI USB controller.
+        # This line should be removed if the legacy DWC2 controller is required
+        # (e.g. for USB device mode) or if USB support is not required.
+        otg_mode=1
```

Comment:
This might be problematic, depending on the CM4 usage. Is it a good idea to outright apply this?

Reply:
I think the vast majority of users will want this, and the minority who don't can still override it.